eclass/eutils.eclass

```diff
@@ -20,17 +20,22 @@ _EUTILS_ECLASS=1
 # implicitly inherited (now split) eclasses
 case ${EAPI:-0} in
 0|1|2|3|4|5|6)
-	inherit desktop epatch estack ltprune multilib preserve-libs \
-		toolchain-funcs vcs-clean
+	inherit desktop edos2unix epatch estack l10n ltprune multilib \
+		preserve-libs toolchain-funcs vcs-clean wrapper
+	;;
+7)
+	inherit edos2unix l10n wrapper
 	;;
```

Comment:
add
`*)     die "${ECLASS}.eclass is banned in EAPI ${EAPI}" ;;`
for EAPI >= 8